core/commands/object/object.go

```diff
@@ -21,10 +21,10 @@ import (
 	ft "github.com/ipfs/go-ipfs/unixfs"
 )
 
-// ErrObjectTooLarge is returned when too much data was read from stdin. current limit 512k
-var ErrObjectTooLarge = errors.New("input object was too large. limit is 512kbytes")
+// ErrObjectTooLarge is returned when too much data was read from stdin. current limit 2m
+var ErrObjectTooLarge = errors.New("input object was too large. limit is 2mbytes")
 
-const inputLimit = 512 * 1024
```

Comment:
nitpick: I want to move towards using `2 << 20` for things like this. The go team does this on all go projects and it makes it really easy to read, ex:

```
1 << 10 == 1K
4 << 10 == 4K
2 << 20 == 2MB
4 << 30 == 4GB
```